scripts/_common.sh

```diff
@@ -1,12 +1,24 @@
 #!/bin/bash
 
+#=================================================
+# COMMON VARIABLES
+#=================================================
+
+# dependencies used by the app
+pkg_dependencies="libav-tools|ffmpeg"
+
+YNH_PHP_VERSION="7.3"
+
+extra_php_dependencies="php${YNH_PHP_VERSION}-mysql php${YNH_PHP_VERSION}-curl php${YNH_PHP_VERSION}-simplexml php${YNH_PHP_VERSION}-gd php${YNH_PHP_VERSION}-ldap"
+
 #=================================================
 # EXPERIMENTAL HELPERS
 #=================================================
 
 # Execute a command with Composer
 #
-# usage: ynh_composer_exec --phpversion=phpversion [--workdir=$final_path] --commands="commands"
+# usage: ynh_composer_exec [--phpversion=phpversion] [--workdir=$final_path] --commands="commands"
+# | arg: -v, --phpversion - PHP version to use with composer
 # | arg: -w, --workdir - The directory from where the command will be executed. Default $final_path.
 # | arg: -c, --commands - Commands to execute.
 ynh_composer_exec () {
```

Comment:
If we use the already packaged zip of ampache we can remove this

Reply:
if we use the packaged zip contain all the needed, yes the whole composer part can be removed